components/pipeline-service/staging/base/kustomization.yaml

```diff
@@ -9,7 +9,6 @@ commonAnnotations:
 
 resources:
   - https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=2be0e3a49809ba66bf64625d01833de90c457094
-  - chains-signing-secrets.yaml
```

Comment:
this was already missing in https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/production/base/kustomization.yaml 

is that expected @Roming22 ?

Reply:
Yes. As part of this PR I moved the config from `staging/base` to `base`.

Reply:
Locally I also regenerated the `deploy.yaml` for staging to check that they were not impacted.

Reply:
sgtm thanks